pkgs/tools/text/uwuify/default.nix

```
@@ -0,0 +1,27 @@
+{ lib, fetchFromGitHub, rustPlatform, ... }:
```

Please avoid using `...` in derivations, we prefer to explicitly specify everything we want to hand through to the derivation. (Exception exist of course)

```suggestion
{ lib, fetchFromGitHub, rustPlatform, stdenv, libiconv }:
```

See below for explanation.